### DISCUSSION TITLE
forge 1.19.3

### DISCUSSION THREAD

**Discussion by apollodrift** [ORelio/Minecraft-Console-Client#2439] [General]
so i went into config and everything did the "force" for forge and when i try to connect it says something like clien on net 2 this server is on net 3. if someone could please help it would be really apriciated. thank you!

**Comment by milutinke** [comment]
Forge does not work above 1.17/8 (Not exactly sure which version), it definitely does not work in 1.19 and above.
There is now FML 3 protocol and no documentation on it, also this is primarily a vanilla client, so it's not a priority.
